main/keychain/src/main/java/org/cryptomator/keychain/KeychainAccessStrategy.java

```diff
@@ -5,7 +5,7 @@
  *******************************************************************************/
 package org.cryptomator.keychain;
 
-interface KeychainAccessStrategy {
+public interface KeychainAccessStrategy {
```

Comment:
Is there a reason for this change? I mean, it is not bad,  I just wonder.

Reply:
Changed in 92321a7